### DISCUSSION TITLE
Dynamic filter removed on page back

### DISCUSSION THREAD

**Discussion by amit-meshram** [Budibase/budibase#9200] [Help] [answered]
My Badibase app is using dynamic filter to filter records from table. Each record is editable. If I select any filtered record to edit and come back on page, applied filter is getting removed. Is there any way to persist the filter state? or any alternative option to achieve this requirement?

**Comment by melohagan** [comment, marked as answer]
Hey @amit-meshram 

At the moment there is no way to bind Dynamic Filter values, and no way to persist the values.
